Dockerfile

```diff
@@ -1,19 +1,18 @@
-FROM node:14.19.0-alpine3.15
+FROM ubuntu:20.04
 
-RUN apk update
-RUN apk upgrade
+RUN apt-get update \
+  && apt-get install -y curl make g++ \
+  && curl -sL https://deb.nodesource.com/setup_14.x | bash \
+  && apt-get install -y nodejs \
```

Comment:
Which version of node we are installing here?

Reply:
https://deb.nodesource.com/setup_14.x

Reply:
@Jayachand did we try with ubuntu 22.04?

Reply:
As we are see memory increase with ubuntu 20.04, just a suggestion, should we try with 22.04 and see?